invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml

```diff
@@ -74,7 +74,7 @@
     </datafield>
     <datafield tag="538" ind1=" " ind2=" ">
       <subfield code="a">Use this with a Micro-CernVM available at</subfield>
-      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+      <subfield code="w">http://cernvm.cern.ch/portal/downloads</subfield>
```

Comment:
Yes, but you should keep URLs in $u and not change it to $w as your PR does... that's what I meant in the above comment.